SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -0,0 +1,167 @@
+//==------- ctor_default.cpp  - DPC++ ESIMD on-device test ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu || level_zero
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// Test for esimd default constructor.
+
+#include <CL/sycl.hpp>
+#include <iostream>
+#include <string>
+#include <sycl/ext/intel/experimental/esimd.hpp>
+
+#include "../../../esimd_test_utils.hpp"
+#include "../logger.hpp"
+#include "../type_coverage.hpp"
+
+using namespace cl::sycl;
+using namespace sycl::ext::intel::experimental;
+using namespace sycl::ext::intel::experimental::esimd;
+using namespace esimd_test::api::functional::type_coverage;
+using namespace esimd_test::api::functional::logger;
```

Comment:
```suggestion
```
Do you really need a separate `type_coverage` namespace or a `using namespace logger` expression here?

Reply:
this comment was applied in this commit: 3787c4fc6d62817f7c4e1458c2e5255a60beeef4